src/components/Form/Controls/EditInPlace/EditInPlace.test.tsx

```diff
@@ -112,41 +121,30 @@ describe("EditInPlace", () => {
     expect(asFragment()).toMatchSnapshot();
 
     const input = getByRole("textbox");
-    expect(input).toBeInvalid();
-
-    // The message hasn't showed up yet
-    expect(queryByText("you did not fill this")).not.toBeInTheDocument();
-
-    await act(async () => {
-      // Focus and submit the form
-      await userEvent.keyboard("{tab}{enter}");
-    });
+    act(() => expect(input).toBeInvalid());
```

Comment:
This looks like the wrong way to use `act` - it should wrap mutations, not expectations

https://react.dev/reference/react/act#await-act-async-actfn

Reply:
Yet this expectation does fire an `invalid` event on the form internally, which causes the component to react and re-render. I don't yet understand whether this is considered a bug and if so, which Testing Library repo I should file it on, but I should at least add a comment here.

Reply:
that is awful everything =(